KONDUTKODEKSO.md

```diff
@@ -0,0 +1,25 @@
+# Kondutkodekso
+
+Ni estas kognaj.  
+Esti kogna estas esti limhava.  
+En nia limigo ni faras elektojn, kiuj estas malsaĝaj aŭ estas misaj.
+
+Se ni faras malsaĝajn elektojn pro nia limigeco,  
+ni ne rajtas juĝi aliaj pro la sama kialo.
+
+Do, ni ne povas juĝi,  
+tial ni pardonas.
+
+Tiu ĉi projekte kaj ĝiaj rezultaĵoj estas intencata kiel:  
+loko de lernado,  
+loko de kompreno,  
+loko de instruado,  
+loko de komunigo,  
+loko, kie kreantoj kreas la ilojn por helpi al aliaj kreantoj krei komplikajn aĵojn elegante.
+
+Fartu bone, kreanto. Fartu bone kaj kreu.
+
+---
+
+Surbaze de la [Kodekso de Kreanto v2](https://github.com/Xe/creators-code).
+Bonvolu legi la ligitan retpaĝon por pli da informoj.
```

Comment:
The English original asks to "read the link", but thereby implies reading the content that's being linked to, not the link itself, so I've translated this as "la ligitan retpaĝon" (the linked webpage).